### PR TITLE
(docs) add info to decode OIDC token

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
Link to [**preview**](https://deploy-preview-3977--gitpod-kumquat.netlify.app/docs/configure/workspaces/oidc#openid-connect-oidc-in-gitpod)

---

|Before|After|
|:---:|:---:|
|image|image|

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3977"></a>

### CODE REVIEW

File: src/content/docs/configure/workspaces/oidc.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ $ gp idp token --audience example.org
 eyJhbGciOiJSUzI1NiIsImtpZCI6ImlkLTE2ODQ3NTc4MDY...
 ```
 
-The output, when decoded, translates to a JSON object similar to the one below.
+The output, when decoded (using any JWT decoder), translates to a JSON object similar to the one below.
```
Comment (loujaybee) [inline src/content/docs/configure/workspaces/oidc.md:59]: **Suggestion:** Can we give an example here? This feels like it's not so helpful without an example? The user will read this, and then have to go to google to figure this out?

**Comment by Siddhant-K-code** [comment]
> **@akosyakov**: not sure what was motivation but looks ok

Currently. `gp idp token` returns a JWT token which is not human readable. And, there is an extra step the user needs to perform to decode to make it usable information. For future, we can have a optional flag (e.g., `--decode`) in CLI to directly decode and return the JSON output. And, currently, there was no instruction regarding how to decode.

File: src/content/docs/configure/workspaces/oidc.md (hunk 1 above)
Comment (Siddhant-K-code) [reply src/content/docs/configure/workspaces/oidc.md]: Resolved in [`f07c7dc`](https://github.com/gitpod-io/website/pull/3977/commits/f07c7dc2d5cdc2f6c4c16ed0385f7a2128f0bd84)

**Review by loujaybee — APPROVED** [review]

**Comment by loujaybee** [comment]
Thanks @Siddhant-K-code, a theoretically small tweak, but could make a difference to our developer experience—thank you !

**Pull request merged**